ca_on_toronto/jurisdiction.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+from utils import CanadianJurisdiction
+
+from .constants import (
+    TWO_LETTER_ORG_CODE_SCHEME,
+    )
+
+class TorontoJurisdiction(CanadianJurisdiction):
+    def __init__(self):
+        super(TorontoJurisdiction, self).__init__()
+
+    def get_organizations(self):
+        for org in super(TorontoJurisdiction, self).get_organizations():
```

Comment:
If the supermethod ever yields other organizations, you won't want to make them all "City Council".